project-base/.travis.yml

```
@@ -19,7 +19,7 @@ jobs:
             script:
                 - sudo apt install ruby
                 - gem install docker-sync
-                - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                - sed -i -r "s#sed -i \'\' -E#sed -i -r#" ./scripts/install.sh
```

```suggestion
                - sed -i -r "s#sed -i '' -E#sed -i -r#" ./scripts/install.sh
```
Escaping is wrong here